formatting/colors/color.go

```diff
@@ -78,7 +78,7 @@ func Parse(value interface{}) Color {
 	case 'g', "g", "minecoin_gold", MinecoinGold:
 		return MinecoinGold
 	default:
-		return White
+		return Gray
```

Comment:
this my first time working with Go so not really sure what make sense to return here to indicate that a color was provided that wasn't expected. Realistically this probably won't happen?

just made the change to `Gray` to leave a comment here; can change it back if need be

Reply:
I approve this change for now, but I will likely be modifying this method after merging to include some sort of okay-bool to let the user know if the value was parsed successfully.